experimental/dispatch_profiler/profile_all.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Sets up a venv suitable for running IREE Dispatch Profiler and executes
+# a suite of runs. This is invoked by a Github workflow and can be invoked
+# locally.
+#
+# Recommend getting default 'python' to be python 3. For example on Debian:
+#   sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+# Or launch with python=/some/path
+
+set -euo pipefail
+
+TD="$(cd $(dirname $0) && pwd)"
+
+PYTHON="${PYTHON:-python3}"
+
+DISPATCH_PROFILER_IREE_BIN_DIR=${1:-""}
+if [ -z ${DISPATCH_PROFILER_IREE_BIN_DIR} ]; then
```

Comment:
nit: It's recommended to use `[[]]` (https://google.github.io/styleguide/shellguide.html#s6.3-tests)

```suggestion
if [[ -z "${DISPATCH_PROFILER_IREE_BIN_DIR}" ]]; then
```